### DISCUSSION TITLE
Getting an error about a listing being "frozen" when trying to update it

### DISCUSSION THREAD

**Discussion by seandmccarthy** [etsy/open-api#1096] [Q&A]
One of our merchants is getting this error:

"This listing is not publishable because it's frozen." 

when we're trying to use updateListing on an existing listing in draft (edit) state.

What can we do to detect this state before attempting to run an update? Where is the "frozen" state surfaced, and what does it mean?

Thanks

**Comment by sushansht** [comment]
Yes i think returning status of shop in shop object in getShop or getShopByUserId will solve the problem. 
Please Etsy could you add this in the api?
